Sources/KlaviyoSwift/KlaviyoAPI.swift

```diff
@@ -70,7 +70,8 @@ struct KlaviyoAPI {
 
         if httpResponse.statusCode == 429 {
             requestRateLimited(request)
-            return .failure(KlaviyoAPIError.rateLimitError)
+            let retryAfter = httpResponse.allHeaderFields["Retry-After"] as? Int
```

Comment:
may not affect ios, but on android this response headers' value was a map of lists of strings, `Map<String, List<String>>`

Reply:
Were you able to test that this way to parse `Retry-After` header does work on live 429 response from production?

Reply:
I didn't have a chance to test it out last night but tested now and it was indeed not returning the header value. Switched to using `value(forHTTPHeaderField:` which works.